spec.bs

```diff
@@ -2162,9 +2159,19 @@ of the following global objects:
     1. Let |realm| be the result of [=creating a new script runner realm=] given
       {{InterestGroupScoringScriptRunnerGlobalScope}}.
     1. Let |browserSignalsJS| be |browserSignals| [=converted to ECMAScript values=].
-    1. TODO: convert |trustedScoringSignals| to a JS value.
+    1. Let |convertedTrustedScoringSignals| be a new empty [=map].
+    1. [=map/For each=] |key| -> |value| of |trustedScoringSignals|:
+      1. Let |valueString| be the result of [=serializing an Infra value to a JSON string=] given
+        |value|.
+      1. If the previous steps did not [=exception/throw=] an exception:

```

Comment:
Can the previous steps throw? It doesn't look like it. https://infra.spec.whatwg.org/#serialize-an-infra-value-to-a-json-string calls two algorithms, and both use the `!` short-hand before calling them so I think they're defined to never ever throw. See https://tc39.es/ecma262/#sec-returnifabrupt-shorthands.

But it feels like we're using these as a way to bypass WebIDL conversion logic, which feels wrong. Can we just use WebIDL's convert instead of relying on these infra algorithms? Or at least some clarification on why one was chosen over the other would be good.